packages/commitlint-config-jira/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { UserConfig } from '@commitlint/types';
+
+import { releases } from './ingores';
```

Comment:
typo in the file path `ingores`

```suggestion
import { releases } from './ignores';
```

Reply:
🤦 will fix